src/Illuminate/Foundation/Testing/Concerns/InteractsWithAuthentication.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Illuminate\Foundation\Testing\Concerns;
+
+trait InteractsWithAuthentication
+{
+    /**
+     * Return true if the user is authenticated, false otherwise.
+     *
+     * @param  sring|null $guard
```

Comment:
Sorry to be a pain. Could you please put exactly 2 spaces before the dollar in each case.